pkg/cli/admin/release/info.go

```diff
@@ -734,16 +742,34 @@ func (i *ReleaseInfo) Platform() string {
 	return fmt.Sprintf("%s/%s", os, arch)
 }
 
-func (o *InfoOptions) LoadReleaseInfo(image string, retrieveImages bool) (*ReleaseInfo, error) {
+// LoadReleaseInfo takes an image and returns release payload info or an error. The release image will be looked up using
+// alternative image sources from image content source policy objects and/or from the user-given string, if alternateImageRef
+// is true. If retrieveImages is true, release payload component images will also be fetched.
+func (o *InfoOptions) LoadReleaseInfo(image string, retrieveImages bool, alternateImageReference bool) (*ReleaseInfo, error) {
+	replacedRef := false
+	opts := extract.NewExtractOptions(o.IOStreams)
+	opts.SecurityOptions = o.SecurityOptions
+	opts.FileDir = o.FileDir
 	ref, err := imagesource.ParseReference(image)
 	if err != nil {
 		return nil, err
 	}
+	fromContext, err := opts.SecurityOptions.Context()
+	if err != nil {
+		return nil, err
+	}
+	newRef, repo, _, err := fromContext.PreferredImageSource(ref.Ref, o.SecurityOptions.AlwaysTryAlternativeSources, o.SecurityOptions.Insecure)
```

Comment:
This goes against the whole point of hiding this inside the registryclient interfaces (hiding the detection of failover).  This is exactly the sort of change I didn't want to make in oc - all we should have to do is identify the input ICSP and provide that to the context, and then everything else should work automatically with no change.  Doing it here defaults the point.

Reply:
I've updated both PRs, ptal